### DISCUSSION TITLE
arbitrary values with comma not work

### DISCUSSION THREAD

**Discussion by pincman** [tailwindlabs/tailwindcss#8612] [Help]
```css
@apply tw-shadow-[-10px_0_15px_-3px_#fcd34d,-4px_0_6px_-4px_#fcd34d]; 
```
it not work

**Comment by rohmann** [comment]
Hi @pincman, I tried this out in Tailwind Play (latest Tailwind version) and it seems to be working just fine with the latest version. Here are some links with contrasting shadows using both utility classes and the `@apply` technique:

https://play.tailwindcss.com/2WNN2RSfbP
https://play.tailwindcss.com/PnzELBiBwf

Maybe it's something in your environment or PostCSS setup?

**Reply by tkozuch** [reply]
Hi. I know it's an old issue, but seems like for Tailwind 3, this still is an issue (checked on play.tailwindcss.com).

The shown class is translated to:

```
.shadow-\[-10px_0_15px_-3px_\#fcd34d\2c -4px_0_6px_-4px_\#fcd34d\] {
  --tw-shadow: -10px 0 15px -3px #fcd34d,-4px 0 6px -4px #fcd34d;
  --tw-shadow-colored: -10px 0 15px -3px var(--tw-shadow-color), -4px 0 6px -4px var(--tw-shadow-color);
  box-shadow: var(--tw-ring-offset-shadow, 0 0 #0000), var(--tw-ring-shadow, 0 0 #0000), var(--tw-shadow);
}
```

(note the "\2c " in place of comma)

the troubling thing about this is the space character which breaks the CSS

is there any workaround for this in Tailwind 3?

**Reply by wongjn** [reply]
The space is part of the escape sequence, it works in [Tailwind Play](https://play.tailwindcss.com/oYtKTLn6YC).
